build.gradle.kts

```diff
@@ -1,3 +1,4 @@
+val junitVersion = "5.5.2"
```

Comment:
`val junitVersion by extra { "5.5.2" }`
in subprojects refer to it as
`val junitVersion: String by rootProject.extra`